### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by Rulox**
Update docs before creating the release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,8 +8,26 @@ Version: 0.1.0
...
+`GET` - Get all logged users orders. If the user is not logged in, it returns nothing.
+`POST` - Create an order using the current user's cart. It returns the created order or an error.
+
+##### `/ecommerce/checkout`
+`POST` - It requires `order_id` parameter in URL. This endpoint tries to make the payment of the
+order passed as parameters. 
+
+**Note:** Please read the following considerations.
+
+    1. The user has to be the owner of the order, or it should be a guest order.
```
Comment (mitogh) [inline README.md]: Can you remove the indentation from the list so the text uses a ordered list.

File: README.md [hunk 2]
```diff
@@ -8,8 +8,26 @@ Version: 0.1.0
...
 
+---
 ### Endpoints
-TODO
+##### `/ecommerce/cart`
+`GET` - Get the current user's cart.
+`POST` - Add a product to the user's cart. `product_id` needs to be specified as an url parameter.
+
+##### `/ecommerce/order`
+`GET` - Get all logged users orders. If the user is not logged in, it returns nothing.
```
Comment (mitogh) [inline README.md]: `returns nothing` you mean `returns empty array` ? or what `nothing means in this context?

File: README.md (hunk 1 above)
Comment (Rulox) [inline README.md]: Yeah I don't know what happened there sorry

**Comment by Rulox** [comment]
You can have a look now, thank you!

**Pull request merged**